doc/user/content/sql/delete.md

```diff
@@ -21,7 +21,12 @@ _alias_ | Only permit references to _table_name_ as _alias_.
 
 ## Details
 
-`DELETE` cannot currently be used inside [transactions](../begin).
+### Known limitations
+
+* `DELETE` cannot be used inside [transactions](../begin).
+
+* **Low performance.** While processing a `DELETE` statement, Materialize cannot
+  process other `INSERT`, `UPDATE`, or `DELETE` statements.
```

Comment:
I think the one caveat to add to this (and all others), is that it's only true when the `DELETE` involves a read. That's almost all deletes so it's not a big deal, but maybe worth calling out.

Reply:
Sorry for the delay on this. @jkosh44, I don't understand what you mean by a delete involving a read. Can you explain that further?

Reply:
I'm also confused. I think all `DELETE`s today involve a read?

Reply:
Yes, you're right, I was confusing myself. You can ignore this.

Reply:
Realized we didn't have an issue for `RETRACT`/`DELYEET`, so I filed one: https://github.com/MaterializeInc/materialize/issues/19071